felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/EventTypes.kt

```diff
@@ -22,7 +22,8 @@ enum class BehovType {
     PERSISTER_SAK_ID,
     OPPRETT_OPPGAVE,
     PERSISTER_OPPGAVE_ID,
-    HENT_IM_ORGNR
+    HENT_IM_ORGNR,
```

Comment:
Bruk past tense for events ikke verb. Dette navnet er OK for behov ikke for event.